sormas-ui/src/main/java/de/symeda/sormas/ui/task/TaskEditForm.java

```diff
@@ -104,6 +114,14 @@ protected void addFields() {
 		taskContext.setImmediate(true);
 		taskContext.addValueChangeListener(event -> updateByTaskContext());
 
+		final HorizontalLayout saveInfoLayout = new HorizontalLayout(
```

Comment:
This hint is currently always displayed, even when entering a task from the task directory